packages/ui/src/theme/overwrites/components/input.ts

```diff
@@ -0,0 +1,32 @@
+export const Input = {
+  sizes: {
+    sm: {
+      field: {
+        padding: '6px 12px',
+        borderRadius: '6px',
+      },
+    },
+    md: {
+      field: {
+        padding: '8px 16px',
+        borderRadius: '6px',
+      },
+    },
+    lg: {
+      field: {
+        padding: '10px 16px',
+        borderRadius: '6px',
+      },
+    },
+  },
+  variants: {
+    outline: {
+      field: {
+        _focus: {
+          background: '#FFF',
```

Comment:
We shouldn't use colors that aren't included in the theme. Here you can safely write `white` as Chakra has such value in its default theme.